ts/DataGrid/Table/Actions/RowsVirtualizer.ts

```diff
@@ -244,17 +256,30 @@ class RowsVirtualizer {
      */
     private renderRows(rowCursor: number): void {
         const { viewport: vp, buffer } = this;
-        const rowsPerPage = Math.ceil(
+        const isVirtualization = this.rowSettings?.virtualization;
+        const rowsPerPage = isVirtualization ? Math.ceil(
             vp.tbodyElement.offsetHeight / this.defaultRowHeight
-        );
+        ) : Infinity; // need to be refactored when add pagination
 
         const rows = vp.rows;
 
+        if (!isVirtualization && rows.length > 50) {
+            console.warn(
```

Comment:
I like this idea ❤️

Reply:
What if the user wants to make a very tall DataGrid that will contain more than 50 rows? In this case it is good to turn off virtualization, because those 50 rows will be rendered anyway. That is why I wonder if it is a better idea to use a number equal to e.g. `tableHeight / defaultRowHeight + 50` instead of `50`.

Reply:
However, it's only a warning and we do not change the rendering way.

Reply:
Yes, but this warning will not always be justified. E.g. here: https://jsfiddle.net/tqnza8y4/